src/framework/core/js/Fluid.js

```diff
@@ -826,6 +826,28 @@ var fluid = fluid || fluid_3_0_0;
         return isFinite(string) && ((string % 1) === 0) ? Number(string) : NaN;
     };
 
+    /**
+     * Derived from AGK's stack overflow answer ( http://stackoverflow.com/a/12830454 )
+     *
+     * Rounds the supplied number to at most the number of decimal places indicated by the scale, omiting any trailing 0s.
+     * If the scale is invalid (i.e falsey, not a number, negative value), it is treated as 0.
+     * If the scale is a floating point number, it is rounded to an integer.
+     *
+     * @param {number} num - the number to be rounded
+     * @param {number} scale - the maximum number of decimal places to round to.
+     * @return {number} The num value rounded to the specified number of decimal places.
+     */
+    fluid.roundToDecimal = function (num, scale) {
+        // treat invalid scales as 0
+        scale = scale && scale >= 0 ? Math.round(scale) : 0;
+        var number = Math.round(num * Math.pow(10, scale)) / Math.pow(10, scale);
```

Comment:
We should tidy up this code a bit by hoisting out the repeated values - I recommend definitions like var scalep = Math.pow(10, scale), and then replace Math.pow(10, scale + 1) with simply 10 * scalep